open_fdd/ecm_engineering/honesty_export.py

```
@@ -0,0 +1,303 @@
+        (5, "Model_Provenance", "IDF / EPW / G14 / cascade timestamps"),
+        (6, "Inputs", "Named building inputs; yellow FLH = FITTED_FROM_EPLUS (not validation)"),
+        (7, "Industry_Screening", "Independent ESCO methods (2nd set of eyes)"),
+        (8, "Measures", "Primary roster + Savings_Check; status FITTED/BALLPARK/NO_EP/FAIL_SIGN"),
+        (9, "Demand", "July weekday / weekend / load-shed kW when twin payload present"),
+        (10, "Twin_Calibrate", "Optional monthly twin calibration when attached"),
+        (12, "Status meanings", ""),
+        (13, "FITTED", "Hours reverse-solved from E+ so sheet≈E+; wiring_echo_pct≈0 is audit only"),
+        (14, "BALLPARK", "Independent screening assumptions; use pct_diff_industry_vs_eplus"),
+        (15, "NO_EP", "No EnergyPlus / patch missing — do not invent ep_kwh"),
+        (16, "FAIL_SIGN", "Sheet vs E+ opposite sign (broken physics / patch)"),
+        (18, "Savings_Check", "Flag measures that exceed ~20–30% of baseline (2nd-eyes)"),
```

Ruff (0.16.0)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)